reconcile/utils/ocm.py

```diff
@@ -388,14 +386,6 @@ def _get_create_cluster_spec(cluster_name: str, cluster: OCMSpec) -> dict[str, A
     def _get_update_cluster_spec(update_spec: Mapping[str, Any]) -> dict[str, Any]:
         ocm_spec: dict[str, Any] = {}
 
-        instance_type = update_spec.get("instance_type")
-        if instance_type is not None:
-            ocm_spec["nodes"] = {"compute_machine_type": {"id": instance_type}}
-
-        private = update_spec.get("private")
-        if private is not None:
-            ocm_spec["api"] = {"listening": "internal" if private else "external"}
-
```

Comment:
this change is not mentioned in the PR description. can you elaborate?

Reply:
This code wasn't enabled, it was just a copy of the OSD logic and it wasn't called by `update_cluster`. ROSA clusters with STS do not allow changing the privacy once they are created.

Reply:
thanks for the clarification